core/src/main/java/com/linecorp/armeria/server/ServiceConfigBuilder.java

```diff
@@ -320,6 +320,10 @@ ServiceConfig build(ServiceNaming defaultServiceNaming,
                 requestIdGenerator != null ? requestIdGenerator : defaultRequestIdGenerator, errorHandler);
     }
 
+    public ServiceConfigBuilder getPrefixedServiceConfigBuilder(String contextPath) {
```

Comment:
I don't think this method is necessary if we can set the `contextPath` when `ServiceConfigBuilder#build` is invoked.